tests/atest/rules/spacing/variable-should-left-aligned/test.robot

```diff
@@ -0,0 +1,13 @@
+*** Settings ***
+Library   String
+Documentation   test
+
+
+*** Variables ***
```

Comment:
Verify also with only one space - its not recognized as separator but part of the name do it could cause problems (`` ${var} value``)

Reply:
Also add gold cases - correctly aligned to test it doesn't trigger rule